markdown/org/docs/patterns/lunetius/fabric/en.md

```diff
@@ -2,6 +2,7 @@
 title: Fabric options 
 ---
 
+### Main Fabric
```

Comment:
It looks like the "Main Fabric" heading is only ever used on patterns which require more than one fabric:
https://freesewing.org/docs/patterns/cornelius/fabric/
https://freesewing.org/docs/patterns/penelope/fabric/
https://freesewing.org/docs/patterns/sandy/fabric/
https://freesewing.org/docs/patterns/holmes/fabric/

Reply:
Yea I see the point of removing it here since it's a singular fabric. Though on the others it is due to them having different needs dependent on lining etc.